client/client.go

```diff
@@ -259,20 +259,20 @@ type serviceModeKeeper struct {
 	// triggering service mode switching concurrently.
 	sync.RWMutex
 	serviceMode     pdpb.ServiceMode
-	tsoClient       atomic.Value // *tsoClient
+	tsoClient       *tsoClient
```

Comment:
why remove atomic?

Reply:
We have already used sync.RWMutex as shown by line 259. atomic.Value isn't necessary and should be removed last time.